WebApp/src/components/Root.js

```diff
@@ -6,13 +6,15 @@ import { PersistGate } from 'redux-persist/integration/react';
 import { ConnectedRouter } from 'connected-react-router'
 import { MuiThemeProvider, createMuiTheme } from '@material-ui/core/styles';
 import blue from '@material-ui/core/colors/blue';
-import pink from '@material-ui/core/colors/pink';
+import grey from '@material-ui/core/colors/grey';
+import blueGrey from '@material-ui/core/colors/blueGrey';
```

Comment:
maybe just: import {blue, grey, blueGrey} from '@material-ui/core/colors' should be better